README.md

```diff
@@ -32,3 +33,163 @@ zopen build
 Note that running `zopen build` against your development repo will NOT install the code into your
 environment.
 
+
+# Developer Notes
+
+The detailed version of above.
+
+## Workflow
+
+The workflow for contributing to the meta repo.  This workflow assumes
+your installation of zopen is installed in `${HOME}`.
+
+### Step 1 - Update existing meta
+
+This will update the ~/zopen/usr/local/bin versions of the system binaries as part
```

Comment:
`~/zopen/usr/local/bin` is a configurable location. Perhaps you can update it as such:

Step 1:
Update your z/OS Open Tools meta package before proceeding